### DISCUSSION TITLE
Help/Discussion for as to why sub-tabs in folder tabs aren't showing whatsoever anymore?

### DISCUSSION THREAD

**Discussion by SHHSSH** [piroor/treestyletab#3663] [General]
Even though I hated them at times for how they would act visually, I did really appreciate being able to see the sub-tabs in a different layout.
Is this a known malfunction @piroor ?
![image](https://github.com/user-attachments/assets/1442a37b-8c5e-4f0b-b6f3-300e16f66558)

**Comment by irvinm** [comment]
Known issue (https://github.com/piroor/treestyletab/issues/3655) and will be fixed in next release.

**Reply by piroor** [reply]
I uploaded a fixed version and it is still under review...

**Comment by SHHSSH** [comment]
@piroor Did you also fix the visual layout somehow? It seems so much better now! 

Thanks!!!
